test/threads_exec.jl

```diff
@@ -722,15 +722,65 @@ let a = zeros(nthreads())
 end
 
 # static schedule
-function _atthreads_static_schedule()
+function _atthreads_static_schedule(n)
+    ids = zeros(Int, n)
+    Threads.@threads :static for i = 1:n
+        ids[i] = Threads.threadid()
+    end
+    return ids
+end
+@test _atthreads_static_schedule(nthreads()) == [1:nthreads();]
```

Comment:
I don't think we need to "materialize" the array?

```suggestion
@test _atthreads_static_schedule(nthreads()) == 1:nthreads()
```